R/parcel_geomarkers.R

```diff
@@ -7,26 +7,33 @@ d_out <- d |> bind_cols(get_parcel_data(d$address))
 
 # housing violation data
 d_violation <-
-  fs::path("data-raw", "Violations Likley Related to Lead, Asthma, and Mental Health from 2002 to date.xlsx") |>
-  readxl::read_excel() |>
-  filter(as.Date(ISSUED) >= as.Date("2014-01-01") & as.Date(ISSUED) <= as.Date("2021-12-31")) |>
-  mutate(parcel_id11 = substr(PARCEL_NO, 2, 12)) |> # removing one extra zero
-  group_by(parcel_id11) |>
-  summarize(n_violation = n())
+  codec::read_tdr_csv("https://raw.githubusercontent.com/geomarker-io/curated_violations/main/curated_violations/tabular-data-resource.yaml") |>
```

Comment:
let's use the release-specific url to download the files from the github release assets;  for a version I just released, it would be `https://github.com/geomarker-io/curated_violations/releases/download/0.1.2/tabular-data-resource.yaml`